travis-cli/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM ruby:2.4-alpine
+FROM ruby:2.6-alpine
```

Given they don't seem to publish an official recommended/supported Ruby version (or unfortunately even seem to maintain this tool really), this seems fine to me; just as elsewhere, I think we should take the opportunity to pin:

```suggestion
FROM ruby:2.6-alpine3.10
```